wiki/Application-Issues.md

```diff
@@ -1,8 +1,27 @@
+### Electron based applications
+
+Electron based applications can work directly on Wayland, but they don't accept it without configuration. Configuration differs between versions. Information was grabbed from the [ArchWiki](https://wiki.archlinux.org/title/Wayland#Electron).
+
+For Electron (>28) you need to set environment variable:
+```kdl
+environment {
+    ELECTRON_OZONE_PLATFORM_HINT "auto"
```

Comment:
Should this say "wayland" instead?

Reply:
Auto first check wayland, then fallback to x11. I think it is good, if XWayland still present or niri used in combination with other WMs.
https://www.electronjs.org/docs/latest/api/environment-variables#electron_ozone_platform_hint-linux

Reply:
I see. Makes you wonder why it's still not the default

Reply:
Oh, I didn't know that electron apps respect this variable. I copied system `.desktop` files tom my users home to overwrite the `exec` commands, and append `--ozone-platform-hint=auto` instead. This is much better :heart: